### ISSUE TITLE
Default Map features reset every new Rapid Instance

### ISSUE THREAD

**Issue by bradlewt** [facebook/Rapid#906]
### Description

Whenever a new Rapid instance is opened, say from MapRoulette, the image background selection remains consistent with the previous session, but the selected Map features resets every time.  Would it be possible for the map features to likewise remain consistent across instances?

### Version

2.0.1

### What browser are you seeing the problem on? What version are you running?

Firefox v111.0

### The OS you're using

win

### Steps to reproduce

Open task in MapRoulette,
Turn off map feature (e.g. Landcover)
Upload and commit edits
Open next MapRoulette challenge
Observe the Map features you previously turned off are now back on.

### The browser URL at the time you encountered the bug

https://mapwith.ai/rapid#background=Maxar-Premium&datasets=fbRoads,msBuildings&map=16.68/10.08845/15.37601

### The auto-detected useragent string for your browser (leave blank if you're manually filling this form out)

Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/111.0

**Comment by bhousel** [comment]
Yes this makes sense to me.. We should be restoring the user's feature filtering preferences from localStorage if the url hash doesn't specify what features to filter.
